rules/config

```diff
@@ -2,59 +2,63 @@
 ## Configuration parameters for SONiC build system
 ###############################################################################
 
+## Note that this file should be valid for inclusion from both Makefile and shell
+## scripts. No whitespace allowed between assignment (=) sign, shell script
+## compat stuff should go into functions.sh in SNIiC build system root directory.
```

Comment:
>SNIiC [](start = 47, length = 5)

typo

Reply:
I reworked code and leave this file as is: no need to source from shell scripts at the moment.